browser/js/content.js

```diff
@@ -413,3 +442,9 @@ function findParentByClass(el, className) {
   }
   return null;
 }
+
+function replaceChildren(parent, children) {
```

Comment:
Mmm was using it instead of morphdom for a while. I don't really gain anything from DOM diffing but I guess I'll stick with morphdom anyways.